app/src/main/java/com/example/financemanager/data/services/PlannedExpenditureWorkManager.kt

```diff
@@ -0,0 +1,68 @@
+package com.example.financemanager.data.services
+
+import android.Manifest
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import androidx.core.app.ActivityCompat
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import androidx.work.CoroutineWorker
+import androidx.work.WorkerParameters
+import com.example.financemanager.R
+import com.example.financemanager.data.receivers.NotificationReceiver
+
+class PlannedExpenditureWorkManager(
+    private val ctx: Context,
+    private val params: WorkerParameters
+) :
+    CoroutineWorker(ctx, params) {
+    override suspend fun doWork(): Result {
+        val category = inputData.getString("title") ?: ""
+        val description = inputData.getString("description") ?: ""
+        val cost = inputData.getString("cost") ?: ""
+        val bodyString = "$category : $cost\n$description"
+        val id = inputData.getLong("id", 0)
+        notifyPlannedExpenditure(ctx, bodyString, id)
+        return Result.success()
```

Comment:
Neet to add empty line before return, also you can use with (inputData) { val category = getString("title) ?: "" ...}